src/EditorFeatures/TestUtilities/ServicesTestUtilities.csproj

```diff
@@ -94,6 +94,7 @@
     <InternalsVisibleToTest Include="Roslyn.Compilers.VisualBasic.Test.Utilities" />
     <InternalsVisibleToTest Include="Roslyn.InteractiveHost.UnitTests" />
     <InternalsVisibleToTest Include="Roslyn.Services.Editor.UnitTests" />
+    <InternalsVisibleToTest Include="Roslyn.VisualStudio.Test.Utilities" />
```

Comment:
Is VisualStudioTestUtilities the project we intend to make live in a separate repo (eventually)? If so, then perhaps we should avoid IVT now before we make a tangled mess?